backend/danswer/connectors/google_drive/connector.py

```diff
@@ -464,6 +462,7 @@ def _fetch_docs_from_drive(
                             ],
                             source=DocumentSource.GOOGLE_DRIVE,
                             semantic_identifier=file["name"],
+                            doc_updated_at=datetime.fromisoformat(file["modifiedTime"]),
```

Comment:
should we also add a `.astimezone(timezone.utc)` here just for clarity / safety?